src/Transition/CMakeLists.txt

```diff
@@ -68,7 +68,7 @@ foreach(i RANGE 1 ${end})
   set_target_properties("${LAST_NAME}" PROPERTIES FOLDER Auxiliary)
   target_link_libraries("${LAST_NAME}" TransitionLib)
 
-  if("${i}" GREATER "22")
+  if("${i}" GREATER "33")
```

Comment:
This pushes the install to only include "V9-0 to V9-1" and above.  That change alone could make a notable impact in installer size.